CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changes
 =======
 
+Unreleased
+----------
+
+* Now requires ``zyte-common-items >= 0.22.0``.
```

Comment:
should it mention SERP spider?

Reply:
Yes, I just added that bit because I was afraid we could miss it when writing the notes. I don’t think we will miss the spider part :slightly_smiling_face: